networktest.go

```diff
@@ -53,6 +54,11 @@ var (
 	defaultClientPort = 19556
 
 	harnessNetParams = &chaincfg.SimNetParams
+
+	// logOutput is a flag that can be set to append the output from the
+	// seed nodes to log files.
```

Comment:
Comment should be updated to reflect the fact that it lets you target _any_ running node.